README.md

```diff
@@ -2661,6 +2661,23 @@ Other Style Guides
     $sidebar.find('ul').hide();
     ```
 
+  <a name="jquery--attach-events"></a><a name="25.5"></a>
+  - [25.5](#jquery--attach-events) Avoid mess up with the events queue.
```

Comment:
can you elaborate on this a bit more? Is the advice specifically to ignore "off"?